src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -605,6 +605,12 @@ public void updateFeedbackResponseForChangingSection(
             FeedbackResponseAttributes response) throws InvalidParametersException, EntityDoesNotExistException {
 
         FeedbackResponse feedbackResponse = frDb.getFeedbackResponseEntityOptimized(response);
+        if (feedbackResponse == null) {
+            // the response might have already being deleted.
```

Comment:
due to?

Reply:
Updated comments. Is it okay to state it in this way?